--- Qt/5.15.2/gcc_64/qml/QtQuick/Controls/Private/EditMenu_base.qml.orig	2020-12-21 17:30:57.641640241 -0800
+++ Qt/5.15.2/gcc_64/qml/QtQuick/Controls/Private/EditMenu_base.qml	2020-12-21 17:31:11.049873254 -0800
@@ -154,7 +154,7 @@
     Connections {
         target: mouseArea
 
-        function onClicked() {
+        function onClicked(mouse) {
             if (input.selectionStart === input.selectionEnd) {
                 var cursorPos = input.positionAt(mouse.x, mouse.y)
                 input.moveHandles(cursorPos, cursorPos)
